src/CMakeLists.txt

```diff
@@ -35,19 +51,21 @@ if(NOT CMAKE_BUILD_TYPE STREQUAL Debug)
       CMAKE_C_FLAGS_${build_type_upper})
     string(REGEX REPLACE "(^| )[/-]D *NDEBUG($| )" " " "${flags}" "${${flags}}")
   endforeach()
-endif()
+endif(NOT CMAKE_BUILD_TYPE STREQUAL Debug)
```

Comment:
@tchaikov 
I was under the impression that "while there" one is allowed to fix imperfacetions in
code?
But I'll take it out.